index.html

```diff
@@ -0,0 +1,158 @@
+              <a href="https://twitter.com/footballlz"><i class="fab fa-twitter"></i></a>
+            </li>
+            <li><a>English</a></li>
+            <li><a>My page</a></li>
+            <li><a>Logout</a></li>
+          </ul>
+        </div>
+        <div class="navba">
+          <a href="index.html"><img src="images/Capture.png" class="navImage" alt="" /></a>
+          <ul>
+            <li><a href="#">program</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li>
+            <li><a href="./about.html">about</a></li>
+            <li><button type="button">CT Campaign</button></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <main> 
+      <section class="ban">
+        <h1>The 2022 
+            <br>sports
+             <br>celebration</h1>
+        <div class="ban-summary">
+          <p>
+            FIFA is a non-profit organization that describes itself as an international governing body of association football, futsal and beach soccer. It is the highest governing body of association football.
+          </p>
+        </div>
+        <h2>2021.10.12(tue) ~ 15(fri)</h2>
+        <p class="place">@ stellanbosch, South africa</p>
+      </section>
+
+      <section class="prog">
+        <h3>Main Program</h3>
+        <hr class="line" />
+
+        <div class="prog-line">
+          <div id="prog-0" class="cards">
+            <img src="images/calendar.svg" alt="" />
+            <p class="title">Calendar</p>
+            <p>date and time are important  to everyone around the world</p>
+          </div>
+          <div id="prog-1" class="cards">
+            <img src="images/card.svg" alt="" />
+            <p class="title">Card</p>
+            <p>Represent the worth  of every player out there shows how important you are</p>
+          </div>
+          <div id="prog-2" class="cards">
+            <img src="images/folder.svg" alt="" />
+            <p class="title">folder</p>
+            <p>to save protect your info all the time for the near future</p>
+          </div>
+          <div id="prog-3" class="cards">
+            <img src="images/people.svg" alt="" />
+            <p class="title">people</p>
+            <p>communication represent the huminity of a person to the world</p>
+          </div>
+          <div id="prog-4" class="cards">
+            <img src="images/shield.svg" alt="" />
+            <p class="title">shield</p>
+            <p>protect yourself at all time surround yourself with the one who understand you </p>
+          </div>
+        </div>
+        <h3 class="linked"><a href="">SEE THE WHOLE PROGRAM</a></h3>
+        <button type="button">join sports summit program 2021</button>
+      </section>
+
+      <section class="speakers">
+        <h3>featured speakers</h3>
+        <hr class="line" />
+        <div class="speakers-lineout"></div>
```

Comment:
- [ ] Images of the features are currently not loading. It must be a bad referencing problem. Kindly fix the issue.

![image](https://user-images.githubusercontent.com/4247740/164860584-d83dc72c-45dc-46ab-b2de-8e1a3230f49e.png)